frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -351,206 +381,189 @@ export const DetachedTestResultsList = ({
   );
 
   return (
-    <>
-      {detailsModalId && (
-        <TestResultDetailsModal
-          testResultId={detailsModalId}
-          closeModal={() => {
-            setDetailsModalId(undefined);
-          }}
-        />
-      )}
-      <div className="grid-row">
-        <div className="prime-container card-container sr-test-results-list">
-          <div className="sticky-heading">
-            <div className="usa-card__header">
-              <h2>
-                Test results
-                {!loadingTotalResults && (
-                  <span className="sr-showing-results-on-page">
-                    {getResultCountText(
-                      totalEntries,
-                      pageNumber,
-                      entriesPerPage
-                    )}
+    <div className="grid-row">
+      <div className="prime-container card-container sr-test-results-list">
+        <div className="sticky-heading">
+          <div className="usa-card__header">
```

Comment:
Github went pretty wild on the diffs here - am I correct in thinking that the only thing that actually changed was removing the details modal to an if block above this return?

Reply:
Yes that's right the rest of that is just formatting.